python_modules/dagster/dagster_tests/core_tests/definitions_tests/decorators_tests/test_config_mapping.py

```diff
@@ -0,0 +1,96 @@
+import enum
+import re
+
+import pytest
+from dagster import (
+    DagsterInvalidConfigError,
+    Enum,
+    Field,
+    check,
+    op,
+    graph,
+    config_mapping,
+)
+
+def test_empty_config_mapping():
+    @op
+    def empty_op():
+        pass
+
+    @graph
+    def empty_graph():
+        empty_op()
+
+    @config_mapping
+    def empty_config_mapping(_):
+        return {}
+
+    assert empty_graph.to_job(config=empty_config_mapping).execute_in_process().success
+
+@op
+def my_op(context):
+    return context.op_config["foo"]
+
+@graph
+def my_graph():
+    my_op()
+
+def test_bare_config_mapping():
+    @config_mapping
+    def my_config_mapping(val):
```

Comment:
hmm this is a bit freaky. Out of the scope of this PR, but probably worth investigating what's happening under the hood here, make sure there's some type of schema somewhere that's being checked against.

Reply:
Yes, I will look into this